### PR TITLE
Remove sys_conf from sys_log and hardcode sys_conf function into sys_log

### PR DESCRIPTION
**Pull request by javibu13**

### CODE REVIEW

File: code/sys_log/src/system_logger_tool/sys_log.py [hunk 1]
```diff
@@ -183,14 +202,14 @@ def sys_log_logger_get_module_logger(name : str,
             else:
                 # For test file. Ex: Test_APP_DIAG
                 name = name_list[0]
-        custom_level = sys_conf_read_config_params(
+        custom_level = sys_log_read_config_params(
```
Comment (mariuscrsn) [inline code/sys_log/src/system_logger_tool/sys_log.py]: remove sys_log, is private

Comment (mariuscrsn) [reply code/sys_log/src/system_logger_tool/sys_log.py]: rename to: `_read_config_params`

**Review by mariuscrsn — CHANGES_REQUESTED** [review]

File: code/sys_log/src/system_logger_tool/sys_log.py [hunk 2]
```diff
@@ -150,6 +152,23 @@ def format(self, record) -> str:
         return formatter.format(record)
 
 
+class SysLogSectionNotFoundErrorC(Exception):
```
Comment (mariuscrsn) [inline code/sys_log/src/system_logger_tool/sys_log.py]: Private exception

**Review by mariuscrsn — CHANGES_REQUESTED** [review]

File: code/sys_log/src/system_logger_tool/sys_log.py (hunk 1 above)
Comment (javibu13) [reply code/sys_log/src/system_logger_tool/sys_log.py]: Done

File: code/sys_log/src/system_logger_tool/sys_log.py (hunk 2 above)
Comment (javibu13) [reply code/sys_log/src/system_logger_tool/sys_log.py]: Done

**Review by mariuscrsn — APPROVED** [review]

**Pull request merged**